pymc3/tests/test_distributions.py

```diff
@@ -2280,18 +2330,23 @@ def test_rice(self):
         )
 
     @pytest.mark.xfail(condition=(aesara.config.floatX == "float32"), reason="Fails on float32")
-    def test_moyal(self):
+    def test_moyal_logp(self):
         self.check_logp(
             Moyal,
             R,
             {"mu": R, "sigma": Rplusbig},
             lambda value, mu, sigma: floatX(sp.moyal.logpdf(value, mu, sigma)),
+            n_samples=-1,
         )
+
+    @pytest.mark.xfail(condition=(aesara.config.floatX == "float32"), reason="Fails on float32")
+    def test_moyal_logcdf(self):
         self.check_logcdf(
             Moyal,
             R,
             {"mu": R, "sigma": Rplusbig},
             lambda value, mu, sigma: floatX(sp.moyal.logcdf(value, mu, sigma)),
+            n_samples=-1,
```

Comment:
Xfailed as expected in https://github.com/pymc-devs/pymc3/pull/4497/checks?check_run_id=2021551266